share/spice/indego_bikeshare_locations/indego_bikeshare_locations.css

```diff
@@ -0,0 +1,63 @@
+.mapview-marker__icn.ddgsi-circle {
```

Comment:
whoops, this needs to be namespaced with `.zci--indego_bikeshare_locations` so we only affect the map pins for this IA